examples/train.py

```diff
@@ -185,7 +191,7 @@ def setup_exps_rllib(flow_params,
         config["train_batch_size"] = horizon * n_rollouts
         config["gamma"] = 0.995  # discount rate
         config["use_gae"] = True
-        config["no_done_at_end"] = False
+        config["no_done_at_end"] = True
```

Comment:
Is this needed?

Reply:
I had an error when setting it to true. It's rare but the bug's still there

Reply:
Argh, okay.